roles/openshift_master/tasks/systemd_units.yml

```diff
@@ -0,0 +1,61 @@
+# This file is included both in the openshift_master role and in the upgrade
+# playbooks.  For that reason the ha_svc variables are use set_fact instead of
+# the vars directory on the role.
+
+- debug: var=openshift_version
+
+- name: Init HA Service Info
+  set_fact:
+    ha_svc_svc_dir: "/usr/lib/systemd/system"
```

Comment:
I feel like there's been some discussion on how we could properly ship all of the unit files in the RPMs.  @sdodson, do you remember anything about that?

I'd like to get this merged today if there aren't any major blockers.